configure.sh

```diff
@@ -1,9 +1,23 @@
+# Essential tooling for compilation
+sudo apt-get update
+sudo apt-get install -y build-essential
+sudo apt-get install -y libboost-all-dev
+sudo apt-get install -y make
+sudo apt-get install -y g++
+
+# This gives us the apt-get-repository command,
+# needed for Python 3.6, which has new language features
+# used in our integration tests
+sudo apt-get install -y python-software-properties
```

Comment:
We also can take out these lines for installing Python 3.6, since we are not using that anymore.